Lib/test/test_unittest/test_async_case.py

```diff
@@ -180,6 +180,57 @@ async def on_cleanup(self):
         test.doCleanups()
         self.assertEqual(events, ['asyncSetUp', 'cleanup'])
 
+    def test_exception_in_async_setup(self):
+        class Test(unittest.IsolatedAsyncioTestCase):
+            def setUp(self):
+                self.assertEqual(events, [])
+                events.append('setUp')
+
+            async def asyncSetUp(self):
+                self.assertEqual(events, ['setUp'])
```

Comment:
`test_exception_in_setup` does not contain such checks. If you want to add them, look at the updated `test_full_cycle`. It is more convenient to define the `expected` at the beginning and then use its slices. If you will add more events, you will only need to update one list and several indices.